src/button-group/__tests__/button-group.test.tsx

```diff
@@ -1,274 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
All tests from this file are copied or re-implemented, no test scenario was excluded.